### DISCUSSION TITLE
Star in exchange for a Star

### DISCUSSION THREAD

**Discussion by the1Riddle** [drknzz/GitHub-Achievements#250] [Show and tell]
Hello everyone,

I would appreciate it if you could take a moment to star my profile repository.
Here is the link: https://github.com/the1Riddle/the1Riddle
Thank you in advance!

**Comment by bettyc0der** [comment]
I starred you Please star me too 
https://github.com/salcedocharlotte/CIS261-Course-Project

**Reply by the1Riddle** [reply]
i have done that already

**Reply by ternera** [reply]
Please star my repository and I will return it: https://github.com/ternera/hacker101-ctf

**Reply by KamBRJC** [reply]
star my back https://github.com/z-nnovation/GitSwitcher

**Comment by Mattral** [comment]
I have starred yours please star mine below too
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Reply by the1Riddle** [reply]
i've done that thanks

**Comment by ternera** [comment]
Please star my repository and I will return it: https://github.com/ternera/hacker101-ctf

**Comment by peterbaccaro** [comment]
Please give me a star for my repo and help me get ⭐️ starstruck ⭐️!

https://github.com/peterbaccaro/design-patterns-python

Much appreciated. Thanks in advance.

**Comment by minKasent** [comment]
Hey everyone! 🌟 Doing Star for Star (S4S) to unlock Starstruck.

Please star: https://github.com/minKasent/crypto_exchange

Reply with your repo link below and I will star back immediately! Thank you! 🚀

**Reply by the1Riddle** [reply]
@minKasent however much yo looking for this, do not spam discussions... a better approach would be to create your own discussion and tell other devs what you wish...

regards

**Comment by the1Riddle** [comment]
hello yall...
Since this discussion is closed, y'all can unsubscribe from it.
This would help avoid spam messages sent after the discussion's purpose is met.

regards,
rezzcode.
